src/feat/pitch-functions.cc

```diff
@@ -774,12 +774,20 @@ int32 OnlinePitchFeatureImpl::NumFramesAvailable(
   if (!input_finished_)
     frame_length += nccf_last_lag_;
   if (num_downsampled_samples < frame_length) return 0;
-  else
-    if (input_finished_ && !snip_edges) {
-      return (int32)(num_downsampled_samples * 1.0f / frame_shift + 0.5f);
+  else {
```

Comment:
please make sure the if-else braces are matched, and any braces with else are on the same line, e.g.
if (foo) {
  x;
} else {
  y;
}
you can use cpplint.py (which we recently moved into misc/maintenance/) to check this type of thing.